ReadMe-FormVer.md

```diff
@@ -31,7 +31,21 @@ repositories {
 }
 ```
 
-Command-line options can be enabled using the `formver` configuration block:
+Additionally, make sure you set Kotlin to use K2 and increase the stack size of the Kotlin Daemon:
```

Comment:
What is the Kotlin daemon in this context? You mean the stack size of a JVM's instance?